pkgs/applications/misc/coursera-dl/default.nix

```diff
@@ -41,6 +41,10 @@ in pythonPackages.buildPythonApplication rec {
       url = "https://github.com/coursera-dl/coursera-dl/pull/789.patch";
       sha256 = "sha256:07ca6zdyw3ypv7yzfv2kzmjvv86h0rwzllcg0zky27qppqz917bv";
     })
+    (fetchpatch {
+      url = "https://github.com/coursera-dl/coursera-dl/pull/839.patch";
```

Comment:
Can we fetch the explicit commit instead of a PR that could change over time?

Reply:
Hhm good point. I should have used the commit as well on the previous 789 patch.

Reply:
I've changes both urls to include commit id